symengine/tests/polynomial/test_cancel.cpp

```diff
@@ -0,0 +1,63 @@
+#include "catch.hpp"
+#include <chrono>
+
+#include <symengine/mul.h>
+#include <symengine/add.h>
+#include <symengine/pow.h>
+#include <symengine/polys/cancel.cpp>
```

Comment:
Is this intended?

Reply:
I think not, I was removing the extra imports in next commit.

Reply:
Why is this here?

Reply:
I am importing this otherwise it is giving error. If you can help me here on guinding what i am missing.